.pre-commit-config.yaml

```diff
@@ -77,14 +77,14 @@ repos:
         types: [text] # necessary to include ChangeLog file
         files: ^(ChangeLog|doc/(.*/)*.*\.rst)
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.931
+    rev: v0.940
     hooks:
       - id: mypy
         name: mypy
         entry: mypy
         language: python
         types: [python]
-        args: []
+        args: [--enable-error-code ignore-without-code]
```

Comment:
You can add it directly to `setup.cfg`. In the `mypy` section add
```
enable_error_code = ignore-without-code
```

Reply:
Ah, that was what I was looking for. Thanks!